spec/api/v2/trade_leads/consolidated_spec.rb

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not really. Magic comment added by emacs since the spec contains utf8 characters. Will remove it.